apps/code-examples/src/app/code-examples/modals/inline-help/modal-demo-modal.component.html

```diff
@@ -1,26 +1,36 @@
-<sky-modal>
-  <sky-modal-header>
-    Modal title
-    <sky-help-inline
-      class="sky-control-help"
-      (actionClick)="onActionClick()"
-    ></sky-help-inline>
-  </sky-modal-header>
-  <sky-modal-content></sky-modal-content>
-  <sky-modal-footer>
-    <button
-      class="sky-btn sky-btn-primary sky-margin-inline-sm"
-      type="button"
-      (click)="instance.save()"
-    >
-      Save
-    </button>
-    <button
-      class="sky-btn sky-btn-link"
-      type="button"
-      (click)="instance.close()"
-    >
-      Close
-    </button>
-  </sky-modal-footer>
-</sky-modal>
+<form [formGroup]="demoForm" (submit)="saveForm()">
+  <sky-modal>
+    <sky-modal-header>
+      Modal title
```

Comment:
I updated this demo to mimic the "basic" demo and then just added help inline to it.